README.md

```diff
@@ -1 +1,34 @@
-# Node-Routing-Test-Practice-
\ No newline at end of file
+# Routing Test with Jest 
+
+## Built with
+
+- nvm latest version
+
+- Jest version 23.2.0
+
+- Express version 4.16.3
+
+- ejs version 2.6.1
+
+- ejs-lint version 0.3.0
+
+- superagent version 3.8.3
+
+- supertest version 3.1.0
+
+## Set Up
+```
+$ npm init
```

Comment:
this is not necessary. this command is used only when you create a project from scratch